lib/fog/libvirt/models/compute/server.rb

```diff
@@ -13,6 +13,10 @@ class Server < Fog::Compute::Server
 
         attribute :cpus
         attribute :cputime
+        attribute :firmware
+        attribute :firmware_features
+        attribute :secure_boot
+        attribute :loader
```

Comment:
Note to myself:
The `loader` element points to a specific boot binary (BIOS/UEFI), while `firmware` refers to the type or particular features used to configure or load the guest VM.

Reply:
I think those could be good comments around the attributes.

To me `loader` implies it's singular, but when I look at the code I see it's a list, so should it be
```suggestion
        attribute :loaders
```

Reply:
The loader is not a list, it's a single tag with multiple attributes.

Reply:
And since we're using the same terminology as Libvirt, I believe there's no need to define "firmware" or "loader" as they should already be familiar to users.

Reply:
But in the code I see it's iterated over which made me think it was a list.
```ruby
loader&.each do |key, value|
  xml.loader(key => value)
end
```

Technically I guess it's a hash, but `loader` still made me expect a string.

Reply:
It's a hash, e.g., `loader: { "secure" => "yes" }`.